addons/account/models/company_onboarding.py

```diff
@@ -0,0 +1,161 @@
+from odoo import _, api, fields, models
+
+
+class AccountCompanyOnboarding(models.Model):
+    _inherit = 'res.company'
+
+    onboarding_account_invoice_closed = fields.Boolean(
+        string='Onboarding account invoice closed',
+        help='Refers to the account invoice onboarding panel closed state.')
+    onboarding_account_invoice_folded = fields.Boolean(
+        string='Onboarding account invoice folded',
+        help='Refers to the account invoice onboarding panel folded state.')
+
+    onboarding_company_done = fields.Boolean('Onboarding company done',
+                                             compute='_compute_onboarding_done')
```

Comment:
please always use 4 char indent (also keep in mind 99 char is allowed)

onboarding_invoice_done -> onboarding_account_invoice_done
onboarding_sample_invoice_sent -> onboarding_account_invoice_ sent